src/algorithms/MathUtil.java

```diff
@@ -60,10 +69,25 @@ public static int abs(int n) {
      * @return reversed number
      */
     public static int reverse(int number) {
```

Comment:
It always returns 0;
Be careful!

Reply:
not use  System.out.print in this method.
print this method in the main() method   
 e.g  System.out.println(reverse(157));